etna/transforms/holiday.py

```diff
@@ -50,7 +50,7 @@ def transform(self, df: pd.DataFrame) -> pd.DataFrame:
         if (df.index[1] - df.index[0]) > datetime.timedelta(days=1):
             raise ValueError("Frequency of data should be no more than daily.")
 
-        cols = df.columns.get_level_values("segment")
+        cols = np.unique(df.columns.get_level_values("segment"))
```

Comment:
Can be changed to `df.columns.get_level_values("segment").unique()`

Reply:
you can use suggestions here
```suggestion
        cols = df.columns.get_level_values("segment").unique()
```